frontend/src/components/Cart.tsx

```diff
@@ -37,6 +41,15 @@ const Cart: React.FC = () => {
             .catch(error => console.log("Error removing product from cart: ", error));
     };
 
+    const handleClearCart =()=>{
+        axios.delete("/api/cart/clear", {withCredentials:true})
```

Comment:
I never save withCredentials before. Are you always reading login data from the session cookie?